content/battery.md

```diff
@@ -160,7 +168,9 @@ sudo ifconfig enp4s0f2 up
 upower -d
 ```
 
-This will show the information that your computer can read about the battery.
+This will show the information that your computer can read about the battery. The output from `upower -d` is also included when generating log files in Settings -> Support -> Create Log Files:
+
+![Create Log Files](/images/battery/create-logs.png)
```

Comment:
The window is unfocused in this screenshot, so the button looks greyed out. Can we get a screenshot with the window focused?